scripts/Run-Angular-Locally.ps1

```diff
@@ -9,4 +9,4 @@ $ErrorActionPreference = 'Stop'
 
 . $PSScriptRoot\Custom-Hive.ps1
 
-Test-Template "angular" "angular" "Microsoft.DotNet.Web.Spa.ProjectTemplates.2.2.0-preview3-t000.nupkg" $true
+Test-Template "angular" "angular" "Microsoft.DotNet.Web.Spa.ProjectTemplates.2.2.0-rtm-t000.nupkg" $true
```

Comment:
@ryanbrandenburg should this be 3.0 for master?

Reply:
`3.0.0-alpha1`